mos-platform/neo6502/api/console.c

```diff
@@ -0,0 +1,109 @@
+// Copyright 2024 LLVM-MOS Project
+// Licensed under the Apache License, Version 2.0 with LLVM Exceptions.
+// See https://github.com/llvm-mos/llvm-mos-sdk/blob/main/LICENSE for license
+// information.
+
+#include <string.h>
+#include "../neo6502.h"
+#include "../kernel.h"
+#include "api-internal.h"
+
+__attribute__((leaf))
```

Comment:
Leaf is useless on definitions.

Reply:
Sorry, I kind of grew the habit of putting the attribute in too many places as opposed to too few. Thanks!